config/product/explorer.js

```diff
@@ -237,18 +254,6 @@ export function init(store) {
     exact:            true,
   });
 
-  virtualType({
-    label:            'Namespaces',
-    group:            'cluster',
-    icon:             'globe',
-    namespaced:       false,
-    ifRancherCluster: false,
```

Comment:
Think this should stay, it's the fallback if ifRancherCluster is false

Reply:
Yeah, I don't even remember removing this. Sorry.